lib/puppet-catalog-test/puppet_adapter/puppet_3x_adapter.rb

```diff
@@ -6,9 +6,10 @@ def initialize(config)
       super(config)
 
       require 'puppet/test/test_helper'
+      parser = config[:parser]
 
-      # works 3.7.x
-      if version.start_with?("3.7.")
+      # works 3.x
+      if version.start_with?("3.")
```

Comment:
It looks like this was added in 3.1.0, will update this to 3.1.0+ so the Travis tests pass.

see https://github.com/puppetlabs/puppet/blob/3.1.0/lib/puppet/test/test_helper.rb